sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -550,6 +552,32 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
     }
   }
 
+  test("SPARK-27552: hive.exec.stagingdir is invalid on Windows OS") {
```

Comment:
This test pass even without the change in `getStagingDir`.

Reply:
If it runs on windows, it will fail without this change

Reply:
I see, since the code is not using `File.separator` anymore I'm fine with this.